.ebextensions/nginx.config

```diff
@@ -368,22 +372,14 @@ files:
             return 301 https://$host$request_uri;
           }
 
-          # Expire rules for static content
-
-          location ~* \.(?:css|js)$ {
+          location ~* \.(?:manifest|appcache|xml|json|pdf|docx?)$ {
             access_log off;
-            add_header Cache-Control "max-age=31536000";
+            add_header Cache-Control "max-age=0";
           }
 
-          location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|webp|mp4|ogg|ogv|webm|htc|ttf|ttc|otf|eot|woff|woff2)$ {
+          location /assets/ {
             access_log off;
-            add_header Cache-Control "max-age=2592000";
-          }
-
-          location ~* \.svgz$ {
-            access_log off;
-            add_header Cache-Control "max-age=2592000";
-            gzip off;
+            add_header Cache-Control "max-age=31536000";
```

Comment:
Rather than have complicated file extension matching, heavily cache _everything_ in `/assets`.